_episodes_rmd/05-data-structures-part2.Rmd

```diff
@@ -1,25 +1,27 @@
 ---
 title: "Exploring Data Frames"
-teaching: 20
-exercises: 10
-questions:
-- "How can I manipulate a data frame?"
-objectives:
-- "Be able to add and remove rows and columns."
-- "Be able to remove rows with `NA` values."
-- "Be able to append two data frames"
-- "Be able to articulate what a `factor` is and how to convert between `factor` and `character`."
-- "Be able to find basic properties of a data frames including size, class or type of the columns, names, and first few rows."
 keypoints:
-- "Use `cbind()` to add a new column to a data frame."
-- "Use `rbind()` to add a new row to a data frame."
-- "Remove rows from a data frame."
-- "Use `na.omit()` to remove rows from a data frame with `NA` values."
-- "Use `levels()` and `as.character()` to explore and manipulate factors"
-- "Use `str()`, `nrow()`, `ncol()`, `dim()`, `colnames()`, `rownames()`, `head()` and `typeof()` to understand structure of the data frame"
-- "Read in a csv file using `read.csv()`"
-- "Understand `length()` of a data frame"
+- Use `cbind()` to add a new column to a data frame.
+- Use `rbind()` to add a new row to a data frame.
+- Remove rows from a data frame.
+- Use `na.omit()` to remove rows from a data frame with `NA` values.
+- Use `levels()` and `as.character()` to explore and manipulate factors
+- Use `str()`, `nrow()`, `ncol()`, `dim()`, `colnames()`, `rownames()`, `head()` and
+  `typeof()` to understand structure of the data frame
+- Read in a csv file using `read.csv()`
+- Understand `length()` of a data frame
+objectives:
+- Be able to add and remove rows and columns.
+- Be able to remove rows with `NA` values.
+- Be able to append two data frames
+- Be able to articulate what a `factor` is and how to convert between `factor` and
+  `character`.
+- Be able to find basic properties of a data frames including size, class or type
+  of the columns, names, and first few rows.
+questions: How can I manipulate a data frame?
 source: Rmd
+teaching: 20
+exercises: 10
```

Comment:
I am not sure why you reordered/changed the YAML header here. Removing the quotes from around each of the list elements will cause problems for the parser - can you change this part back to the way it was originally?

Reply:
Hi @naupaka , 
I didn't, at least not intentionally. I don't really work with Rmd files. I think my Rstudio might have made some changes, I'll check the default setup I have. Clearly I changed this somehow though. Apologies!